canonical-web-and-design/maas-ui
# refactor(shared): migrate shared to typescript

## Done

- Update the shared workspace to use typescript.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Run the dev server, everything should run as normal and the header/footer should appear.

## Fixes

Fixes: canonical-web-and-design/maas-ui#1862.
